pages/common/scala.md

```diff
@@ -0,0 +1,19 @@
+# scala
+
+> Scala application launcher and interactive interpreter.
+
+- Call a Scala interactive shell (REPL):
```

Comment:
Perhaps "Launch" is better suited than "Call"

Reply:
"Launch" sounds better to me too. There's no consensus on terms though. [python](https://github.com/hohonuuli/tldr/blob/master/pages/common/python.md) uses "Call", [ruby](https://github.com/hohonuuli/tldr/blob/master/pages/common/ruby.md) uses "Open". The `man scala` page uses "Start". Perhaps we should follow the trend of using a different word for every page ... maybe I should use "Instantiate"? ;-) I'll change it to "Start" to keep it in line with the scala man.